src/industry_gui.cpp

```diff
@@ -447,10 +448,14 @@ class BuildIndustryWindow : public Window {
 						strdim.width = max_minwidth;
 					}
 					d = maxdim(d, strdim);
+
+					if (indsp->grf_prop.grffile != nullptr) {
+						extra_lines_newgrf = max(extra_lines_newgrf, 4);
```

Comment:
Shouldn't this just be
```suggestion
						extra_lines_newgrf = 4;
```
? It's not modified anywhere else (also preferrably with some comment/constant name ? )

Reply:
Done, and added a comment.